bgbb_airflow/pred_airflow_job.py

```diff
@@ -10,19 +10,35 @@
 from bgbb_airflow.bgbb_utils import PythonLiteralOption
 
 
-def extract(spark, ho_start, model_win=90, sample_ids: List[int] = []):
+def pull_most_recent_params(spark):
+    pars_loc = (
+        "s3://net-mozaws-prod-us-west-2-pipeline-analysis/wbeard/bgbb_params/"
```

Comment:
This should be a job parameter (to distinguish between dev and production data).